src/main/resources/com/synopsys/arc/jenkins/plugins/ownership/OwnershipPlugin/floatingBoxTemplate.jelly

```diff
@@ -94,5 +96,14 @@
                 </tr>
             </table>
         </div>
-    </div>
+      </div>
+    </j:if>
+    
+    <!-- Layout for ownership with missing data -->
```

Comment:
There is such option in the global configuration. If it is enabled, <code>isDisplayOwnershipSummaryBox</code> will return false => there will be nothing on the page